web/libs/editor/src/components/TopBar/TopBar.jsx

```diff
@@ -31,44 +30,37 @@ export const TopBar = observer(({ store }) => {
         <Elem name="group">
           <CurrentTask store={store} />
           {store.hasInterface("annotations:view-all") && (
-            <Tooltip title="Compare all annotations">
-              <Button
-                className={"topbar__button"}
-                icon={<IconViewAll width={20} height={20} />}
-                type={isViewAll ? undefined : "text"}
-                aria-label="Compare all annotations"
-                onClick={annotationStore.toggleViewingAllAnnotations}
-                primary={isViewAll}
-                size="medium"
-                style={{
-                  height: 28,
-                  width: 28,
-                  padding: 0,
-                  marginRight: "var(--spacing-tight, 8px)",
-                }}
-              />
-            </Tooltip>
+            <Button
+              className={"topbar__button"}
+              type={isViewAll ? undefined : "string"}
+              aria-label="Compare all annotations"
+              onClick={annotationStore.toggleViewingAllAnnotations}
+              variant={isViewAll ? "primary" : "neutral"}
+              look={isViewAll ? "filled" : "string"}
+              tooltip="Compare all annotations"
+              size="small"
+            >
+              <IconViewAll />
+            </Button>
           )}
           {store.hasInterface("annotations:add-new") && (
             <Tooltip title="Create a new annotation" style={{ "--offset-x": "11px" }}>
```

Comment:
Does it make sense to move this tooltip inside the button as well? That will be consistent. And then you can just remove antd tooltip from this file.